src/transformers/models/greaselm/feature_extraction_greaselm.py

```diff
@@ -0,0 +1,739 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Feature extractor class for GreaseLM."""
+import gzip
+import itertools
+import json
+import os
+import pickle
+from collections import OrderedDict
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Union
+
+import numpy as np
+import torch
+from torch.nn import CrossEntropyLoss
+
+from huggingface_hub import hf_hub_download
+from transformers import AutoModelForMaskedLM, AutoTokenizer
+
+from ...feature_extraction_utils import FeatureExtractionMixin, PreTrainedFeatureExtractor
+from ...utils import TensorType, is_spacy_available, logging, requires_backends
+from ...utils.logging import get_verbosity, tqdm
+from .utils_greaselm import nltk_stopwords
```

Comment:
You can directly include the NLTK stopwords in this file instead of importing it.